env.json

```diff
@@ -1,11 +1,13 @@
 {
-  "BASE_API_URL": "test.polkaswap.io",
```

Comment:
@timofeytrepalin this config is used on Polkaswap.io

but you set it for testnet 

is it okay?